### ISSUE TITLE
Small bug with closing windows in settings

### ISSUE THREAD

**Issue by FrostyMisa** [aicodix/rattlegram#28]
When you open settings and after click any submenu here, like spectrum analyzer or others, when you close it with gesture swipe down, it will open again immediately. Works good when you close it by clicking back button. 
Latest iOS, Rattlegram 1.8. 

I want to thank you for the app, it's really great❤️.

Maybe one small enhancement about spectrum analyzer. It will be nice, if it can be higher with longer history. Like don't limit it only to 2 cm in middle of display. It can continue endlessly down.

**Comment by xdsopl** [comment]
Yes, I've also noticed that swiping doesn't always work reliably for closing sheets on the iOS version. Will see if that can be somehow improved.
A larger waterfall needs more processing and therefore drains the battery faster, which is why I kept it small. Will look if it can be made larger efficiently or at least configurable in both versions.
Thank you for the suggestion.
